jetty-core/jetty-client/src/main/java/org/eclipse/jetty/client/transport/internal/HttpReceiverOverHTTP.java

```diff
@@ -450,9 +450,10 @@ public boolean content(ByteBuffer buffer)
         if (chunk != null)
             throw new IllegalStateException("Content generated with unconsumed content left");
 
-        RetainableByteBuffer networkBuffer = this.networkBuffer;
+        // Retain the chunk because it is stored for later use.
```

Comment:
```diff
--- // Retain the chunk because it is stored for later use.
+++ // Retain the networkBuffer because it is stored as a chunk for later use.
```